wiki/src/js/swim.js

```diff
@@ -69,7 +69,7 @@ function resize()
 
 $(document).ready(function()
 {
-    $('.fishwrap').append("<img src='https://wiki.wetfish.net/upload/52a357b9-3680-9030-34ed-fc68895773c1.png' id='kristyfish'>");
+    $('.fishwrap').append("<img src='src/img/ghostfish.png' id='kristyfish'>");
```

Comment:
This relative path will break on any wiki page that has a slash in it, for example - https://wiki.wetfish.net/news/archives

Reply:
Ohh, yes. I'll have to just do an env lookup for this as well then.

Reply:
Wait, JS, that won't work xD

Reply:
https://github.com/wetfish/classic/pull/38/commits/628078c0acbb29b9d4ad5365ee420d90ca2cff23 should fix, it will pull from the official wiki sources, which shouldn't ever be down in a situation where this will matter